packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -455,6 +455,9 @@ class FlutterErrorDetails extends Diagnosticable {
       properties.add(ErrorSpacer());
       informationCollector().forEach(properties.add);
     }
+    properties.add(StringProperty('library', library));
```

Comment:
Don't add these 3.  This is just a complex method so it is hard to tell that `summary` is actually exposed.
silent doesn't need to be exposed and `library` is exposed indirectly.

Reply:
Is this something we could detect and not lint?

Reply:
this class is a special case that is much more complex than the other cases. I think adding an ignore lint comment is fine here.

Reply:
you could also expose 'library' but just with a low level so it doesn't mess up the string output.